testsuite/flambda2-test-list

```diff
@@ -1,19 +1,10 @@
 # excluded test                  | status      | reason for not running / failure
 # ------------------------------------------------------------------------------------------------
   tests/asmcomp                    FAIL
-  tests/asmgen                     FAIL
   tests/backtrace                  FAIL (FIXME)  there is practically no backtrace info
-#  tests/basic                      FAIL (FIXME)  sets.ml has different bytecodes (ocamlc.byte vs ocamlc.byte) - the digest of the main module differ
-  tests/basic-modules              FAIL (FIXME)  PR#8948 not merged yet in trunk
-  tests/float-unboxing             FAIL (FIXME)  'float_subst_boxed_number.ml' see flambdatest/mlexamples/float_unboxing.ml for a simplified error example
-  tests/gc-roots                   FAIL (FIXME)  test_stdlabels.ml has different bytecodes (ocamlc.byte vs ocamlc.byte) - the digest of the main module differ
-  tests/int64-unboxing             FAIL (FIXME)  'test.ml' (unboxing of recursive continuation parameter)
-  tests/lib-dynlink-init-info      FAIL (FIXME)  fails on macOS
-  tests/lib-dynlink-initializers   FAIL (FIXME)  temporarily disabled (flambda2)
-  tests/lib-dynlink-pr4839         FAIL (FIXME)  temporarily disabled (flambda2)
-  tests/lib-hashtbl                FAIL (FIXME)  htbl.ml has different bytecodes (ocamlc.byte vs ocamlc.byte) - the digest of the main module differ
-  tests/lib-stdlabels              FAIL (FIXME)  test_stdlabels.ml has different bytecodes (ocamlc.byte vs ocamlc.byte) - the digest of the main module differ
-  tests/lib-threads                FAIL (FIXME)  beat.ml seems to fail under heavy load (macOS box on GitHub CI)
-  tests/opaque                     FAIL          'test.ml' (cmx file loading)
+  tests/float-unboxing             FAIL (FIXME)  'float_subst_boxed_number.ml' see flambdatest/mlexamples/float_unboxing.ml for a simplified error example. Should be fixed with unboxing in to_cmm
+#  tests/lib-dynlink-init-info      FAIL (FIXME)  fails on macOS
+#  tests/lib-threads                FAIL (FIXME)  beat.ml seems to fail under heavy load (macOS box on GitHub CI)
   tests/statmemprof                FAIL          Stack traces differ
   tests/warnings                   FAIL          'w55.ml' (@inline attribute), 'w59.ml' (missing warnings when using Obj functions)
+  tests/flambda                    FAIL          'specialize.ml' not implemented in flambda2
```

Comment:
Maybe the other tests in this directory can be run?

Reply:
The others pass just fine. I'm not aware of a way to exclude just one test. Can it be done by appending the file name ?

Reply:
Fixed by making the test pass.